### ISSUE TITLE
Error: No vairant remained!

### ISSUE THREAD

**Issue by gaochengPRC** [choishingwan/PRSice#176]
Hi Sam,

When I implement PRSice, I encountered the error shown below. I have looked through previous issues, unfortunately, didn't find answer to my problem. Can you tell me how to figure it out??

Thank you very much!

Cheng 
```
PRSice 2.2.11.b (2019-10-16)
https://github.com/choishingwan/PRSice
(C) 2016-2019 Shing Wan (Sam) Choi and Paul F. O'Reilly
GNU General Public License v3
If you use PRSice in any published work, please cite:
Choi SW, O'Reilly PF.
PRSice-2: Polygenic Risk Score Software for Biobank-Scale Data.
GigaScience 8, no. 7 (July 1, 2019)
2019-12-14 22:42:19
./PRSice_linux \
    --all-score  \
    --bar-levels 1e-08,5e-08,1 \
    --base base \
    --binary-target T \
    --clump-kb 250 \
    --clump-p 1.000000 \
    --clump-r2 0.100000 \
    --cov chr22.GRCh38.QCC4.tt.pc.eigenvec \
    --cov-col PC1,PC2,PC3,PC4,PC5,PC6,PC7,PC8,PC9,PC10 \
    --fastscore  \
    --or  \
    --out /local/kuiz_grp/PRS/PRS_out/chr22.GRCh38.QCC4.tt.PRS \
    --pheno pheno_tt.txt \
    --seed 807014238 \
    --stat OR \
    --target chr22.GRCh38.QCC4.tt \
    --thread 1

Initializing Genotype file: chr22.GRCh38.QCC4.tt (bed)

Start processing base
==================================================

Reading 100.00%
Base file: base
192125 variant(s) observed in base file, with:
1 NA stat/p-value observed
192124 total variant(s) included from base file

Loading Genotype info from target
==================================================

1548 people (0 male(s), 0 female(s)) observed
1548 founder(s) included

1 variant(s) not found in previous data
25662 ambiguous variant(s) excluded
0 variant(s) included

Error: No vairant remained!

Error:
Execution halted
```

**Comment by choishingwan** [comment]
Strange, the number doesn't match, usually PRSice is rather alright when it
comes to report the reason why some SNPs are missing. I suspect maybe all
of your SNPs are ambiguous in your target file. Could you please check if
there are a total of 25663 variants in your target?

**Comment by gaochengPRC** [comment]
Thank you for your reply!
There are 192125 variants in target file, also 192125 variants in base file. I don't know why this occurs.
Can I send my data to you via email? Then, maybe, you'll find the reason.

**Comment by gaochengPRC** [comment]
Since my plink binary files are transformed from VCF files, no rs number assigned to SNPs, only a dot symbol. So, I replace those dot symbols by chr#_BP_Allele1 (e.g. 22_10527038_G). I don't think that causes the reported error.

**Comment by choishingwan** [comment]
Do send me the files so that I can check. Have you also make sure the
chr#_BP_Allele1 naming is consistent between the base and target file?

Dr Shing Wan Choi
Postdoctoral Fellow
Genetics and Genomic Sciences
Icahn School of Medicine, Mount Sinai, NYC

**Comment by choishingwan** [comment]
The reason behind the problem is because the A2 column in your base file contains a special character, leading to mismatch between the base and target. I will update a fix to try and see if we are capable of removing special characters and also output the mismatch info.

**Comment by choishingwan** [comment]
The above mentioned function is now implemented in ca1426a9dcdf91f46666aaafcba43d7a42a5ea83

However, as I am preoccupied with other work at the moment, I won't be able to provide the new static release anytime soon. You might want to compile PRSice on your own or wait for the next release (which hopefully will come out before the end of year)
